src/Nethermind/Nethermind.AccountAbstraction.Test/Network/CalculateRequestIdTest.cs

```diff
@@ -12,9 +12,12 @@ namespace Nethermind.AccountAbstraction.Test.Network
     [TestFixture]
     public class CalculateRequestIdTest
     {
+        private const ulong ChainId = 5;
```

Comment:
it's not a test that relies on goerli.
it just happens to use a transaction from a chainId with and id of 5.

so since goerli was being limited i wanted to limit reference only to places where  it had to be used.